Move `atmos-gitops-config.yaml` to `atmos.yaml` (#7)
cloudposse/github-action-atmos-terraform-drift-remediation: README.md

          atmos-gitops-config-path: ./.github/config/atmos-gitops.yaml    
```

### Migrating from `v1` to `v2`

> 
> `v2` works only with `atmos >= 1.63.0`
> 
> `v2` drops `install-terraform` input because terraform is not required for affected stacks call
> 
> `v2` drop `atmos-gitops-config-path` input and  `./.github/config/atmos-gitops.yaml`  config. Now you have to use GitHub actions inputs and `atmos.yaml` to specify that configuration.

Reviewer: ```suggestion

The notable changes in `v2` are:

- `v2` works only with `atmos >= 1.63.0`

- `v2` drops `install-terraform` input because terraform is not required for affected stacks call

- `v2` drops `atmos-gitops-config-path` input and the `./.github/config/atmos-gitops.yaml` config file. Now you have to use GitHub Actions environment variables to specify the location of the `atmos.yaml`.
```

Author: Done

> The following configuration fields now moved to GitHub action inputs with the same names

Reviewer: ```suggestion
The following configuration fields moved to the GitHub Action inputs with the same names. Alternatively, you can set `ATMOS_VERSION` or `ATMOS_CONFIG_PATH` environment variables.
```

Author: That's the wrong statement. GHA will override `ATMOS_CONFIG_PATH ` env var.

|         name            |
|-------------------------|
| `atmos-version`         |
| `atmos-config-path`     |

> The following configuration fields are now moved to `atmos.yaml` 

Reviewer: ```suggestion
The following configuration fields moved to the `atmos.yaml` configuration file.
```

Author: Done

> |              name        |    path in `atmos.yaml`                         |

Reviewer: ```suggestion
|              name        |    YAML path in `atmos.yaml`                         |
```

Author: Done

|--------------------------|-------------------------------------------------|
| `aws-region`             | `integrations.github.gitops.storage.region`     | 
| `terraform-state-bucket` | `integrations.github.gitops.storage.bucket`     |
| `terraform-state-table`  | `integrations.github.gitops.storage.table`      |
| `terraform-state-role`   | `integrations.github.gitops.storage.role`       |
| `terraform-plan-role`    | `integrations.github.gitops.role.plan`          |
| `terraform-apply-role`   | `integrations.github.gitops.role.apply`         |
| `terraform-version`      | `integrations.github.gitops.terraform-version`  |
| `enable-infracost`       |  `integrations.github.gitops.infracost-enabled` |
| `sort-by`                |  `integrations.github.gitops.matrix.sort-by`    |
| `group-by`               |  `integrations.github.gitops.matrix.group-by`   |

> If you want `v2` having the same behaviour as `v1` you should  have 

Reviewer: ```suggestion
For example, to migrate from `v1` to `v2`, you should have something similar to the following in your `atmos.yaml`:
```

Author: Done

`./.github/config/atmos-gitops.yaml`
```yaml

> ...

Reviewer: ```suggestion
# ... your existing configuration
```

Author: Done

integrations:
  github:
    gitops:
      terraform-version: 1.5.2
      infracost-enabled: false
      storage:
        region: us-east-2
        bucket: cptest-core-ue2-auto-gitops

>         table: cptest-core-ue2-auto-gitops-plan-storage

Reviewer: Planfile

        role: arn:aws:iam::xxxxxxxxxxxx:role/cptest-core-ue2-auto-gitops-gha
      role:
        plan: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
        apply: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
      matrix:
        sort-by: .stack_slug
        group-by: .stack_slug | split("-") | [.[0], .[2]] | join("-")

> ...

Reviewer: ```suggestion
```

```

`.github/workflows/main.yaml`
```yaml
- name: Remediate Drift
  uses: cloudposse/github-action-atmos-terraform-drift-remediation@v2
  with:
    issue-number: ${{ github.event.issue.number }}
    action: remediate
    atmos-config-path: ./rootfs/usr/local/etc/atmos/  
``` 

> same behaviour as

Reviewer: ```suggestion
This corresponds to the `v1` configuration (deprecated) below.
```

Author: Done

> `./.github/config/atmos-gitops.yaml`

Reviewer: ```suggestion
The `v1` configuration file `./.github/config/atmos-gitops.yaml` looked like this:
```

Author: Done

```yaml
atmos-version: 1.45.3
atmos-config-path: ./rootfs/usr/local/etc/atmos/
terraform-state-bucket: cptest-core-ue2-auto-gitops
terraform-state-table: cptest-core-ue2-auto-gitops
terraform-state-role: arn:aws:iam::xxxxxxxxxxxx:role/cptest-core-ue2-auto-gitops-gha
terraform-plan-role: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
terraform-apply-role: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
terraform-version: 1.5.2
aws-region: us-east-2
enable-infracost: false
sort-by: .stack_slug
group-by: .stack_slug | split("-") | [.[0], .[2]] | join("-")  
```

> 

Reviewer: ```suggestion
And the `v1` GitHub Action Workflow looked like this.
```

Author: Done
